pyne/utils.py

```diff
@@ -216,31 +216,20 @@ def remove(path):
 
 def str_to_unicode(s):
     """
-    This function convert a str from binary or unicode to str (unicode).
-    If it is a list of string, convert every element of the list.
-
-    Parameters:
-    -----------
-    s : str or list of str
-
-    Returns:
-    --------
-    s : text str or list of unicode str
+    Convert str to unicode str.
```

Comment:
We generally require more documentation in the docstring

Reply:
Added in [6f281cd](https://github.com/pyne/pyne/pull/1472/commits/6f281cd77d546cb21dd31fdf34fe1cba39f7af58)
This docstring provides a clear explanation of what the function does, what arguments it takes, what it returns, and what exceptions it may raise. It also provides examples of how the function works in different cases.